src/main/resources/archetype-resources/silo/src/test/java/silo/configuration/TokenCacheTestConfiguration.java

```diff
@@ -0,0 +1,36 @@
+#set($symbol_pound='#')
+#set($symbol_dollar='$')
+#set($symbol_escape='\' )
+package ${package}.${artifactId}.configuration;
+
+import com.rebuy.library.security.cache.TokenCache;
+import com.rebuy.library.security.client.TokenDto;
+import com.rebuy.library.security.model.SecurityToken;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+import java.util.UUID;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@Configuration
+@Profile("testing")
+public class TokenCacheTestConfiguration
+{
+    public final static UUID systemToken = UUID.randomUUID();
+
+    @Bean
+    public TokenCache tokenCache()
```

Comment:
I don't have a strong opinion on how to mock this, @hendrik-rebuy . If you could point to an example, I'd be more than happy to test it and put it into this PR :)